packages/gcf-utils/src/gcf-utils.ts

```diff
@@ -101,6 +115,13 @@ export interface CronPayload {
   cron_org: string;
 }
 
+export enum TriggerType {
```

Comment:
I'm only exporting this for testing purposes. I could not export it and then use either `rewire` or just strings to replicate these values but that seems more fragile. Is this acceptable the way it is? Any other ideas?